front_end/server/lib/routes/servers.js

```diff
@@ -1,4 +1,19 @@
 const express = require('express');
 const router = express.Router();
 const Server = require('../models/server');
-const bodyParser = require('body-parser').json();
\ No newline at end of file
+const bodyParser = require('body-parser').json();
+
+router
+  .get('/', (req, res, next) => {
+    Server.find()
+      .lean()
+      .then(servers => res.send(servers))
+      .catch(next);
+  })
+  .post('/', bodyParser, (req, res, next) => {
+    new Server(req.body).save()
```

Comment:
Here we're creating a new Server document and saving to the DB. The save() mongoose method is promisified, and by default returns the document that was just saved, so in the callback we send that new document back in the server response (we don't have to do this).